green_cli/tx.py

```diff
@@ -0,0 +1,207 @@
+import json
+import logging
+import os
+
+import click
+
+import greenaddress as gdk
+
+from . import context
```

Comment:
We should be consistent here, either all relative or all absolute imports

Reply:
Yes, fixed thanks